search/tests/test_engines.py

```diff
@@ -15,9 +15,10 @@
 from elasticsearch.helpers import BulkIndexError
 from mock import patch
 
-from search.api import perform_search, NoSearchEngineError
+from search.api import NoSearchEngineError, perform_search
 from search.elastic import RESERVED_CHARACTERS
-from search.tests.mock_search_engine import MockSearchEngine, json_date_to_datetime
+from search.tests.mock_search_engine import (MockSearchEngine,
+                                             json_date_to_datetime)
```

Comment:
It looks like this repo doesn't have our standard `isort` configuration yet, which includes bumping the line length to 120: https://github.com/edx/edx-cookiecutters/blob/master/python-template/%7B%7Bcookiecutter.placeholder_repo_name%7D%7D/setup.cfg#L1-L5